epub34/authoring/index.html

```diff
@@ -3109,854 +3108,921 @@ <h3>Parsing URLs in the package document</h3>
 					document as <var>base</var>.</p>
 			</section>
 
-			<section id="sec-shared-attrs">
-				<h3>Shared attributes</h3>
-
-				<p>This section provides definitions for shared attributes (i.e., attributes allowed on two or more
-					elements).</p>
+			<section id="sec-vocab-assoc">
+				<h3>Vocabulary association mechanisms</h3>
 
-				<section id="attrdef-dir" data-epubcheck="true"
-					data-tests="https://w3c.github.io/epub-structural-tests/#05-package-document_package-document.feature_L24">
-					<h4>The <code>dir</code> attribute</h4>
+				<section id="vocab-assoc-intro">
+					<h3>Introduction</h3>
 
-					<div class="note">
-						<p>The <code>dir</code> attribute currently does not have sufficient <a
-								href="https://www.w3.org/policies/process/#adequate-implementation">implementation
-								experience</a>, as [=reading systems=] typically only support a single default
-							directionality for metadata display.</p>
-						<p>Although it lacks the necessary reading system support, it is integral to the content model
-							on which EPUB is built (i.e., for internationalization support in the [=package document=]).
-							Consequently, it is strongly encouraged to set the proper directionality of text values in
-							the package document to ensure proper rendering once this situation improves.</p>
-						<p>The lessening of requirements for this attribute was only done to account for the different
-							process under which EPUB was developed prior to being brought into W3C. New features
-							developed under W3C processes will not receive a similar exemption.</p>
-					</div>
+					<p>EPUB defines a method of referencing terms and properties defined in metadata and semantic
+						vocabularies using the <a href="#sec-property-datatype"><var>property</var> data type</a>. The
```

Comment:
```suggestion
						vocabularies using the <a href="#sec-property-datatype"><var>property</var> data type</a>. The
```

Rereading all that tells me that the "property data type" is not the optimal term. We say that a "property data type" is used in the property attribute, but also in other attributes, which makes it confusing to me. Would it be possible to change the "property data type" to something else?

It could be `shortened URL`, `abbreviated URL`... The [RDF Turtle spec](https://www.w3.org/TR/rdf12-turtle/#dfn-prefixed-name) uses the term `prefixed name` which we could reuse after all, or use `prefixed value` for what it is. I am sure there are other alternatives.

This is a more complex editorial change if you agree with me... sorry about that!

Reply:
It was supposed to be a CURIE data type, but that would have been picking sides in the semantic wars between RDFa and microdata. We landed on property because it was primarily for expressing metadata properties in the package document.

Do you think we could just use CURIE now? The ebnf for the syntax was lifted from RDFa core, although we've since changed the reference to IRIs to use the URL spec.

Reply:
That war will never be over 😀… But both RDFa and microdata are hardly ever used, I am afraid, so it does not matter too much (using json-ld data embedded in the header of the html file won over both). This also means that I do not think people would particularly care, so we could use CURIE without too many problems. 

The downside is that I do not believe anybody will ever update the RDFa Core standard, so if some bugs come up, we are stuck. We may be better off by using either the term CURIE or "compact uri" in our spec, but not touching our own ebnf definitions.

Reply:
> We may be better off by using either the term CURIE or "compact uri" in our spec

Is that like generic v brand name? 😄 

I'd be fine with just calling it a compact url. (cool kids don't use IRIs!) I think somewhere we mention compatibility with rdfa and curies, so that gives us a little distance to justify our own definition.

Reply:
To avoid overloading this pull request, I'm going to merge the current rearrange and open separate PRs to rename the property data type and tweak the introduction.

Reply:
I just wanted to propose that:-)